naga/tests/in/atomicTexture-int64.wgsl

```diff
@@ -0,0 +1,12 @@
+@group(0) @binding(0)
+var image: texture_storage_2d<r64uint, read_write>;
+
+@compute
+@workgroup_size(2)
+fn cs_main(@builtin(local_invocation_id) id: vec3<u32>) {
+    imageAtomicMax(image, vec2<i32>(0, 0), 1lu);
+
+    workgroupBarrier();
+
+    imageAtomicMin(image, vec2<i32>(0, 0), 1lu);
```

Comment:
What about non-atomic reads? Does it work, especially on platforms where it's emulated as Rg32Uint?

Reply:
seems to work... maybe?